docs/howtos/DataPlane Generation - DPG/09versioning.mdx

```diff
@@ -0,0 +1,207 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Versioning
+
+This doc details what emitters will generate for versioned specs
+
+## Single API Version
+
+If there is just one API version in the spec, we will generate
+
+## Multiple API version
+
+The configuration flag `api-version` allows you to toggle the behavior that our emitters will generate.
+
+We will get the versioning information from the `Versions` enum that you define and link as part of the `@typespec/versioning` library.
```

Comment:
What is "link as part of the `@typespec/verisoning` library" referring to?

Reply:
it means the enum you pass to `@versioned`. Updating doc, thanks!